src/hotspot/cpu/x86/interp_masm_x86.cpp

```diff
@@ -1176,28 +1176,35 @@ void InterpreterMacroAssembler::remove_activation(
          Address(rbp, frame::interpreter_frame_sender_sp_offset * wordSize));
 
   if (state == atos && InlineTypeReturnedAsFields) {
+    // Check if we are returning an inline type and load its fields into registers
```

Comment:
Suggestion:
// Check if we are returning an non-null inline type and load its fields into registers

Reply:
Thanks, fixed.